### ISSUE TITLE
`npm install` has error

### ISSUE THREAD

**Issue by SnowCait** [actions/javascript-action#290]
## Issue

I have an error when execute `npm install`.

```
ncc: Version 0.31.1
ncc: Compiling file index.js into CJS
Error: error:0308010C:digital envelope routines::unsupported
```

## Resolve

Bump #283

## Workaround

```
npm install -g npm-check-updates
ncu --upgrade
npm install
```

**Comment by ncalteen** [comment]
Hi @SnowCait! Are you still having this issue? For reference, I have a PR waiting for approval that should fix any issues like this #331

**Comment by SnowCait** [comment]
@ncalteen Awesome CI! I can do `npm install` without error and I've closed this issue.
